src/main/gui/main-window.ts

```diff
@@ -195,6 +196,24 @@ const registerEventHandlerPreSetup = (
             })().catch(log.error);
         });
     }
+
+    // Handle filesystem
+    ipcMain.handle('fs-read-file', async (event, path) => fs.readFile(path, { encoding: 'utf8' }));
+    ipcMain.handle('fs-write-file', async (event, path, content) =>
+        fs.writeFile(path, content, { encoding: 'utf8' })
```

Comment:
This doesn't make sense to me. If `content` can be a buffer, then how does that interact with `encoding: 'utf8'`? Will it ignore the encoding unless you pass in a string?

Reply:
Good point. I'll make the options passed through as well